tests/cross-account-associations/main.tf

```diff
@@ -0,0 +1,195 @@
+/*
+  The workflow for cross-account associations is a bit of a dance. There are several things that must
+  take place in the "owner" account, in order for the associations to work properly in the "member"
+  account.
+
+  Owner account:
+  - Create the resolver
+  - Create the resolver rule and/or the query log config
+  - Create the RAM share
+  - Associate the resolver rule and query log config resources to the RAM share
+  - Invite the "member" account to the RAM share as a principal
+
+  Member account:
+  - Accept the invitation to the RAM share
+  - Create the associations for the resolver rule and/or the query log config
+
+  IMPORTANT: The associations in the member account are dependent on *several* resources that are not
+  direct attributes of the association. In order for terraform to properly manage the ordering of create/destroy
+  actions, use `depends_on` for the RAM share accepter, the principal association, and the resource association.
+*/
+
+provider "aws" {
```

Comment:
Not sure if the make pytest commands are supposed to work or not, but if so this has a conflict with tardigrade-ci/tests/terraform_pytest/aws.tf as that file also defines a provider named aws

If not then disregard

Reply:
Yeah, that's one of the issues I was referring to in my comment on the PR earlier. Working that separately. Our tftest implementation needs some updates to better handle arbitrary aliases and profiles.